openedx/core/djangoapps/catalog/utils.py

```diff
@@ -101,18 +104,66 @@ def get_program_types(user=None):  # pylint: disable=redefined-builtin
         return []
 
 
-def get_programs_data(user=None):
-    """Return the list of Programs after adding the ProgramType Logo Image"""
+def _get_program_instructors(courses, program_id):
+    """
+    Returns the list of instructor from cached if cache key exists otherwise
+    iterate over the courses and return all the instructors of each course run
+    """
+    catalog_integration = CatalogIntegration.current()
+    if catalog_integration.enabled:
```

Comment:
We can remove this check but we need current CatalogIntegration object because we need cache_ttl(cache time to live) value from it.